worker/mails/mail.go

```diff
@@ -53,17 +54,26 @@ func SendMail(ctx *job.TaskContext) error {
 			replyTo = reply
 		}
 	}
+
+	var cfgPerContext map[string]interface{}
+	if opts.Mode == mail.ModeCampaign {
+		cfgPerContext = config.GetConfig().CampaignMailPerContext
+	} else {
+		cfgPerContext = config.GetConfig().MailPerContext
+	}
+
 	ctxName := ctx.Instance.ContextName
-	cfgPerContext := config.GetConfig().MailPerContext
 	if ctxConfig, ok := cfgPerContext[ctxName].(map[string]interface{}); ok {
 		if host, ok := ctxConfig["host"].(string); ok && host != "" {
+			host, _ := ctxConfig["host"].(string)
```

Comment:
I don't understand this line. Is it a copy-paste error?

Reply:
ah yes, I changed my mind on where to store the campaign mailer config and I left this line.